### DISCUSSION TITLE
Can't request GET or POST method using requests or other packages in M1 mac environment

### DISCUSSION THREAD

**Discussion by myeonggilju** [apache/airflow#33246] [General]
### Apache Airflow version

2.6.3

### What happened

I use MLflow provider and slack provider and so on.
I want to request get or post method using PythonOperator or SlackOperator

But It was maintained as running state forever when I request http or https protocol.
It generate well in kafka and database operator
Also, It generate well in docker with linux/amd64 platform environment.
I think It have problem in M1 mac

### How to reproduce

You make python operator and request google site on GET method using requests package
```python
def task1(**context):
    import requests
    res = requests.get("https://google.com")
    print(res.text)
```

### Operating System

ventura 13.1

### Versions of Apache Airflow Providers

apache-airflow==2.6.3
apache-airflow-providers-slack==7.3.1
apache-airflow-providers-http==4.5.0

### Deployment

Virtualenv installation

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

- M1 mac environment
<img width="1087" alt="스크린샷 2023-08-09 오후 2 46 10" src="https://github.com/apache/airflow/assets/96040229/e498975e-f647-42b2-ba36-a6d780be1659">

- docker with linux/amd64 environment
<img width="1457" alt="스크린샷 2023-08-09 오후 2 50 25" src="https://github.com/apache/airflow/assets/96040229/db9699c1-aa7d-4fee-9ac6-5830b51c5835">

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Reply by potiuk** [reply]
You should only open issues if you have good reason to believe and some evidences that you have an airflow issue to report. Otherwise open discussion - where someone might be able to help you in troubleshoot your problem (if they decide to spend their personal time on it and help you for free).

You need to enter the docker and try to reproduce it running requests.get there and see what you get.  You provided screenshots that tell nothing, - some output. But what output do you expect? What's wrong? How it's wrong? Why you have the error? There is nothing to be seen.

Remember that this forum is for people who help others in their free time. If you need help to solve your troubleshooting, you need help those people to help you. investigate it first -  try run it in your different dockers. See what you get when it works when you run it in python, see what you get when it deos not. That will increase your chance on someone willing to spend their time on helping you and actually being able to provide any advices.
